splitio/engine/filters/base_filter.py

```diff
@@ -0,0 +1,28 @@
+import abc
```

Comment:
you are creating `__init__.py` file and is empty, is it correct? I think that you can define the interface in `__init__.py` and avoid to create this one.

Reply:
will do